integration_tests/features/Reorgs.feature

```diff
@@ -30,5 +30,62 @@ Feature: Reorgs
     And I mine a block on B at height 4 with an invalid MMR
     Then node B is at tip BTip1
 
-
-
+@reorg
+  Scenario: Pruned mode reorg past horizon
+    Given I have a base node NODE1 connected to all seed nodes    
+    When I mine a block on NODE1 with coinbase CB1
+    Given I have a base node NODE2 connected to node NODE1
+    When I mine but don't submit a block BLOCK2 on NODE2
+    When I mine but don't submit a block BLOCK3 on NODE2
+    When I mine but don't submit a block BLOCK4 on NODE2
+    When I mine but don't submit a block BLOCK5 on NODE2
+    When I mine but don't submit a block BLOCK6 on NODE2
+    When I mine but don't submit a block BLOCK7 on NODE2
+    When I mine but don't submit a block BLOCK8 on NODE2
+    When I mine but don't submit a block BLOCK9 on NODE2
+    When I mine but don't submit a block BLOCK10 on NODE2
+    When I mine but don't submit a block BLOCK11 on NODE2
+    When I mine but don't submit a block BLOCK12 on NODE2
+    When I mine but don't submit a block BLOCK13 on NODE2
+    When I mine but don't submit a block BLOCK14 on NODE2
+    When I mine but don't submit a block BLOCK15 on NODE2
+    When I mine but don't submit a block BLOCK16 on NODE2
+    When I mine but don't submit a block BLOCK17 on NODE2
+    When I mine but don't submit a block BLOCK18 on NODE2
```

Comment:
Aren't all of these blocks mined at height 2? Did this test pass?

Reply:
No, what happens here, is NODE2 accepts these blocks and it moves forward in the chain.  This just stops the block from propagating.